domdiv/main.py

```diff
@@ -373,6 +407,29 @@ def parse_opts(cmdline_args=None):
         "--exclude-landmarks",
         action="store_true",
         help="Group all 'Landmark' cards across all expansions into one divider.")
+    group_select.add_argument(
+        "--only-type-any", "--only-type", "--type-any",
+        nargs="*",
+        action="append",
+        dest="only_type_any",
+        help="Limit dividers to only those with the specified types. "
+        "A divider is kept if ANY of the provided types are associated with the divider."
```

Comment:
very minor, but missing a space at the end

Reply:
done